app/assets/styles/asylum/main.scss

```diff
@@ -0,0 +1,238 @@
+@charset "utf-8";
```

Comment:
Doesn't matter as per the previous comment about `asylum_scss` bundle. This `main.scss` is not used right now.

The reason this is different is because there are two `.scss` packs in the asylum repo. Like, two separate `main.scss` that seem pretty much the same for some reason. I remember being confused about that too.